File: Modul2/PRAK205-2310817110008-Muhammad_Raihan.py
import math

a,b = map(int, input().split())
c = math.sqrt(pow(b,2)-pow(a,2))
kel = a+b+c
luas = 0.5*c*a
print("Alas = {:g} cm".format(c))
print("Tinggi =  {:g} cm".format(a))
print("Keliling = {:g} cm".format(kel))
print("Luas = {:g} cm^2".format(luas))

File: Modul2/PRAK205-2310817110008-Muhammad_Raihan.c
#include <stdio.h>
#include <math.h>
#define phi 22/7

void main() {
    int A,B,C,keliling,luas;
    scanf("%d %d", &A,&B);
    C=sqrt(pow(B,2)-pow(A,2));
    keliling=A+B+C;
    luas=0.5*C*A;
    printf("Alas = %d cm\n", C);
    printf("Tinggi = %d cm\n", A);
    printf("Keliling = %d cm\n", keliling);
    printf("Luas = %d cm^2\n\n", luas);
}